Plots/Boxplots/NCL_box_3.py

```diff
@@ -0,0 +1,116 @@
+"""
+NCL_box_3.py
+===============
+
+This script illustrates the following concepts:
+   - Drawing box plots
+   - Adding markers to a box plot
+   - Setting the color of individual boxes in a box plot
+   - Setting the width of individual boxes in a box plot
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/box_3.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/box_3_lg.png
+"""
+
+###############################################################################
+# Import packages:
+import numpy as np
+import matplotlib.pyplot as plt
+
+from geocat.viz import util as gvutil
+
+###############################################################################
+# Generate fake data
+
+np.random.seed(200)
+data = np.random.lognormal(size=(40, 3), mean=1, sigma=.7)
```

Comment:
Was there a reason for randomly generating data instead of specifying it manually like in the NCL script? This works so I don't think you have to change it. I'm just curious

Reply:
No reason!